mon-pix/translations/en.json

```
@@ -784,7 +784,7 @@
         "header": "your answers"
       },
       "completion-percentage": {
-        "caption": "progression",
+        "caption": "progression of your customised test",
```

```suggestion
        "caption": "course progression",
```

@AlexNaudyy quelle traduction te semble viable ?

---

Alex a validé ma proposition de traduction